src/ElmFive.elm

```diff
@@ -181,27 +181,23 @@ translations =
 
 translate : String -> String
 translate language =
-    case Dict.get language translations of
-        Nothing ->
-            "five"
-
-        Just translation ->
-            translation
+    Dict.get language translations
+        |> Maybe.withDefault "five"
 
 
 isFive : Int -> Bool
-isFive value =
-    value == five
+isFive =
+    (==) five
 
 
 filter : List Int -> List Int
-filter values =
-    List.filter (\value -> isFive value) values
+filter =
+    List.filter isFive
 
 
 map : List Int -> List Int
-map values =
-    List.map (\value -> five) values
+map =
+    List.map (always five)
```

Comment:
Reasonable. There are some good reasons not to use `always` as it does eager evaluation and can lead to some problematic issues.